src/BitcoinHelpers.js

```diff
@@ -523,6 +523,55 @@ const BitcoinHelpers = {
 
       return transaction.toRaw().toString("hex")
     },
+    /**
+     * Finds all transactions containing unspent outputs received
+     * by the `bitcoinAddress`.
+     *
+     * @param {string} bitcoinAddress Bitcoin address to check.
+     *
+     * @return {Promise<Array>} A promise to the found array of transactions.
+     */
+    findAllUnspent: async function(bitcoinAddress) {
+      return await BitcoinHelpers.withElectrumClient(async electrumClient => {
+        const script = BitcoinHelpers.Address.toScript(bitcoinAddress)
+        return BitcoinHelpers.Transaction.findAllUnspentWithClient(electrumClient, script)
+      })
+    },
+    /**
+     * Gets the confirmed balance of the `bitcoinAddress`.
+     *
+     * @param {string} bitcoinAddress Bitcoin address to check.
+     *
+     * @return {Promise<Number>} A promise to the confirmed balance as satoshis.
+     */
+    getBalance: async function(bitcoinAddress) {
+      return await BitcoinHelpers.withElectrumClient(async electrumClient => {
+        const script = BitcoinHelpers.Address.toScript(bitcoinAddress)
+        return (await electrumClient.getBalanceOfScript(script)).confirmed
+      })
+    },
+    /**
+     * Estimates the fee to have a transaction included within the specified
+     * number of blocks.
+     *
+     * @param {number} [includeWithinBlocks=1] - The estimate is based on having
+     *        the transaction included within this number of blocks. Defaults to
+     *        trying to include the transaction within 1 block.
+     *
+     * @return {Promise<number>} The estimated fee to execute the provided
+     *         transaction. Fails if the fee cannot be estimated.
+     */
+    estimateFeePerKb: async function(includeWithinBlocks) {
+      return BitcoinHelpers.withElectrumClient(async electrumClient => {
+        const feePerKb = await electrumClient.getFeeEstimate(includeWithinBlocks || 1)
```

Comment:
Worth noting, this was always returning 0 in my deposit submitter experiments, which is why I ended up always using the minimum relay fee in [the other PR's implementation](https://github.com/keep-network/tbtc.js/pull/38/files#diff-d8016928a1a8bbd777a2a7d57dc8cbbeR181).

Reply:
Yep, I have the same. I thought there will be a difference if I use it against BTC testnet but nope. I will remove this method and use the default ones or the minimum relay fee in my test script.